packages/ui/src/views/repo/pull-request/details/components/conversation/pull-request-panel.tsx

```diff
@@ -335,73 +293,107 @@ const PullRequestPanel = ({
+                    </span>
+                  </Button>
+                )}
+                {!notBypassable && isMergeable && !isDraft && prPanelData.ruleViolation && (
+                  <Option
+                    className="mr-2.5 items-center"
+                    id="checkbox-bypass"
+                    control={
                       <Checkbox
-                        checked={checkboxBypass}
+                        id="checkbox-bypass"
+                        checked={!!checkboxBypass}
                         onCheckedChange={() => {
                           if (typeof checkboxBypass === 'boolean') {
                             setCheckboxBypass?.(!checkboxBypass)
                           }
                         }}
                       />
-                      <span className="text-12 text-primary">Bypass and merge anyway</span>
-                    </Layout.Horizontal>
-                  )}
+                    }
+                    label="Bypass and merge anyway"
+                    labelProps={{
+                      color: 'secondary'
+                    }}
+                  />
+                )}
 
-                  {actions && !pullReqMetadata?.closed ? (
-                    <ButtonWithOptions
-                      id="pr-type"
-                      theme={buttonState.theme as ButtonWithOptionsTheme}
-                      disabled={buttonState.disabled}
-                      variant={buttonState?.variant as ButtonWithOptionsVariant}
-                      selectedValue={mergeButtonValue}
-                      handleOptionChange={value => setMergeButtonValue(value)}
-                      options={actions.map(action => ({
-                        value: action.id,
-                        label: action.title,
-                        description: action.description
-                      }))}
-                      handleButtonClick={() => {
-                        actions[parseInt(mergeButtonValue)]?.action?.()
-                      }}
-                    >
-                      {actions[parseInt(mergeButtonValue)].title}
-                    </ButtonWithOptions>
-                  ) : (
-                    <Button
-                      theme="primary"
-                      disabled={!checkboxBypass && prPanelData.ruleViolation && !isClosed}
-                      onClick={actions[0].action}
-                    >
-                      Open for review
-                    </Button>
-                  )}
+                {actions && !pullReqMetadata?.closed ? (
+                  <ButtonWithOptions
+                    id="pr-type"
+                    theme={buttonState.theme as ButtonWithOptionsTheme}
+                    disabled={buttonState.disabled}
+                    variant={buttonState?.variant as ButtonWithOptionsVariant}
+                    selectedValue={mergeButtonValue}
+                    handleOptionChange={value => setMergeButtonValue(value)}
```

Comment:
You can probably simplify here
```suggestion
                    handleOptionChange={setMergeButtonValue}
```